R-package/src/xgboost_R.cc

```diff
@@ -38,34 +38,34 @@
 
 using namespace dmlc;
 
-SEXP XGCheckNullPtr_R(SEXP handle) {
+XGB_DLL SEXP XGCheckNullPtr_R(SEXP handle) {
```

Comment:
Why is `XGB_DLL` needed here? Windows users have been using the R package and I don't recall this being an issue.

Reply:
It's already in the header, just matching it so the compiler can check function signature